compiler/passes/resolveIntents.cpp

```diff
@@ -253,12 +254,62 @@ IntentTag blankIntentForExternFnArg(Type* type) {
     return INTENT_CONST_IN;
 }
 
+// Generate an unstable warning if a `const` argument would be transformed into
```

Comment:
This code doesn't generate the warning itself; it generates code to do the checking and potentially warn at runtime. Could you clarify in this comment?

Reply:
Ditto here when you have the time